src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { Modal } from 'react-bootstrap';
-import browser from 'detect-browser';
+const browser = require('bowser');
```

Comment:
@Tamba32 ok, that makes sense. Thanks! I think we should also try to use `import` instead of `require` if possible. Just to keep things consistent, there shouldn't be any difference in performance in this case afaik